gusto/physics.py

```diff
@@ -633,18 +633,19 @@ class InstantRain(Physics):
     timestep dt or over a specified time interval tau.
      """
 
-    def __init__(self, equation, saturation_curve, saturation_variable=None,
+    def __init__(self, equation, saturation_curve, saturation_dependency=None,
                  vapour_name="water_vapour", rain_name=None,
                  convective_feedback=False, set_tau_to_dt=False,
                  parameters=None):
         """
         Args:
             equation (:class: 'PrognosticEquationSet'): the model's equation.
-            saturation_curve (ufl.Expr): the saturation function, above which
-                excess moisture is converted.
-            saturation_variable (str, optional): name of the variable that the
+            saturation_curve (ufl.Expr or :class: `function`): the saturation
+                function, above which excess moisture is converted. Can be
```

Comment:
is it possible to rephrase without using the word "function" or "expression" - just to avoid confusion with the type of the argument...